contrib/storage-jdbc/src/main/java/org/apache/drill/exec/store/jdbc/JdbcStorageConfig.java

```diff
@@ -64,6 +67,10 @@ public String getPassword() {
     return password;
   }
 
+  public boolean areTableNamesCaseInsensitive() {
```

Comment:
Please check ser / de. Maybe you need to specify `@JsonProperty('caseInsensitiveTableNames')` over getter?